erpnext/payroll/doctype/payroll_entry/payroll_entry.py

```diff
@@ -548,35 +548,18 @@ def payroll_entry_has_bank_entries(name):
 	return response
 
 def create_salary_slips_for_employees(employees, args, publish_progress=True):
-	salary_slips_exists_for = get_existing_salary_slips(employees, args)
 	count=0
 	for emp in employees:
-		if emp not in salary_slips_exists_for:
```

Comment:
this condition is required if there is already a salary slip created manually for this employee.